transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -620,9 +620,11 @@ jint JNI_OnLoad_netty_transport_native_epoll(JavaVM* vm, void* reserved) {
     return ret;
 }
 
+#ifndef NETTY_NOT_DYNAMIC
```

Comment:
i picked the negative form because if undefined, the default behavior (dyanmic) should happen.  

Perhaps `NETTY_BUILD_STATIC`?